docs/pages/index.md

```diff
@@ -139,7 +139,7 @@ language: en
       <ul class="section-title">
         {% assign sorted_extensions = site.pages | where: "type","extensions" %}
         {% for item in sorted_extensions %}
-        {% if item.type=="extensions" and item.language=="en" -%}
+        {% if item.type=="extensions" and item.language=="en" and item.abstract -%}
```

Comment:
No. You're addressing the symptom, not the root cause. Add abstracts to all pages.